### ISSUE TITLE
"update failed: Access denied" using meteor object/collection

### ISSUE THREAD

**Issue by eveningkid** [Urigo/angular-meteor#828]
Hey everyone,

I have been playing a bit with angular-meteor before and it worked all well. Today, I just updated Meteor using now the 1.2 version. After a few changes, an issue has unfortunately appeared... :disappointed: 

Basically, I have one collection called `Videos` and I am trying to retrieve the whole collection on a page. This is what I had first: `update failed: Access denied`.

Using:
```javascript
// Server side
Meteor.publish("allVideos", function () {
	return Videos.find();
});
```
```javascript
// Controller
$scope.$meteorSubscribe('allVideos').then(function (handle) {
	$scope.feed = $meteor.collection(Videos, false);
});
```

Then, after allowing update requests on `Videos`, it now says: `update failed: Access denied. In a restricted collection you can only update documents, not replace them. Use a Mongo update operator, such as '$set'.`.

The thing is that I seriously don't know why he needs to update the collection, I just wanted to retrieve the data, nothing more.
If it could help, `insecure` and `autopublish` packages are of course removed.

I really don't know what I have done wrong here...

Any help would really be appreciated! :smile: 

Thank you.

**Comment by CyrilSabbagh** [comment]
Have the same issue ! @Urigo need help :)

**Comment by amor87** [comment]
Im having the same issues, but after updating my project dependencies: here is listed what the meteor upgrade command changed:

accounts-base           upgraded from 1.2.1 to 1.2.2
accounts-oauth          upgraded from 1.1.7 to 1.1.8
accounts-password       upgraded from 1.1.3 to 1.1.4
autoupdate              upgraded from 1.2.3 to 1.2.4
check                   upgraded from 1.0.6 to 1.1.0
ddp-common              upgraded from 1.2.1 to 1.2.2
ddp-server              upgraded from 1.2.1 to 1.2.2
ecmascript              upgraded from 0.1.5 to 0.1.6
ecmascript-collections  removed from your project
ecmascript-runtime      added, version 0.2.6
email                   upgraded from 1.0.7 to 1.0.8
es5-shim                upgraded from 4.1.13 to 4.1.14
less                    upgraded from 2.5.0_3 to 2.5.1
meteor                  upgraded from 1.1.9 to 1.1.10
mongo                   upgraded from 1.1.2 to 1.1.3
promise                 upgraded from 0.5.0 to 0.5.1
random                  upgraded from 1.0.4 to 1.0.5
reactive-dict           upgraded from 1.1.2 to 1.1.3
standard-minifiers      upgraded from 1.0.1 to 1.0.2
templating              upgraded from 1.1.4 to 1.1.5
webapp                  upgraded from 1.2.2 to 1.2.3

Regards

**Comment by Urigo** [comment]
Can you please try `1.0.6`?

**Comment by CyrilSabbagh** [comment]
works for me :) Thanks!

**Comment by eveningkid** [comment]
Works for me too. Is there any Meteor1.2-compatible fix planned soon?

By the way, if someone's having the same issue that we had, go into your `.meteor/versions` file, and edit the line regarding the `angular` package: `angular@1.0.6`. :smiley:

**Comment by amor87** [comment]
It worked great, thanks

**Comment by aaronroberson** [comment]
@Urigo Can you please keep an eye on the gitter chat, this same problem was raised on Oct 29 but has gone unanswered and it cost me an entire day of development.

**Comment by Urigo** [comment]
@aaronroberson you are right, I haven't checked the gitter is a while.
I keep my eyes more on the forums.
I'll make sure to check that out more frequently.
So it looks like I can close this issue

**Comment by aaronroberson** [comment]
I would re-open it actually. The problem is not solved and is still occurring in 1.0.6.

**Comment by Urigo** [comment]
@aaronroberson are you sure you are talking about the same issue everyone else in this thread are talking about?
or maybe it's the issue you linked?
can you share a reproduction?
